Failed IT in cluster mode with scenario tbl

## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?

master

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?

ShardingSphere-Proxy

### Expected behavior

### Actual behavior

### Reason analyze (If you can)

May be related to federal.

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.

maven
```shell
./mvnw -nsu -B install -f shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/pom.xml -Dcheckstyle.skip=true -Dspotless.apply.skip=true -Dit.run.modes=Cluster -Dit.scenarios=tbl -Dit.cluster.adapters=proxy -Dit.cluster.databases=PostgreSQL -Dit.cluster.env.type=DOCKER
```

exception
```shell
[ERROR] Tests run: 328, Failures: 4, Errors: 4, Skipped: 0, Time elapsed: 47.844 s <<< FAILURE! - in org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT
[ERROR] assertExecuteQuery[proxy: tbl -> PostgreSQL -> Literal -> SELECT user_id, CONCAT('SUM:', total, '.') content FROM (SELECT user_id, SUM(order_id_sharding) AS total FROM t_order_federate_sharding GROUP BY user_id HAVING SUM(order_id_sharding) > ?) AS temp](org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT)  Time elapsed: 1.61 s  <<< ERROR!
org.postgresql.util.PSQLException: 
ERROR: syntax error at or near "content"
  位置：44
        at org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT.assertExecuteQueryForStatement(GeneralDQLIT.java:70)
        at org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT.assertExecuteQuery(GeneralDQLIT.java:58)

[ERROR] assertExecute[proxy: tbl -> PostgreSQL -> Literal -> SELECT user_id, CONCAT('SUM:', total, '.') content FROM (SELECT user_id, SUM(order_id_sharding) AS total FROM t_order_federate_sharding GROUP BY user_id HAVING SUM(order_id_sharding) > ?) AS temp](org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT)  Time elapsed: 0.27 s  <<< ERROR!
org.postgresql.util.PSQLException: 
ERROR: syntax error at or near "content"
  位置：44
        at org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT.assertExecuteForStatement(GeneralDQLIT.java:108)
        at org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT.assertExecute(GeneralDQLIT.java:97)

[ERROR] assertExecuteQuery[proxy: tbl -> PostgreSQL -> Placeholder -> SELECT user_id, CONCAT('SUM:', total, '.') content FROM (SELECT user_id, SUM(order_id_sharding) AS total FROM t_order_federate_sharding GROUP BY user_id HAVING SUM(order_id_sharding) > ?) AS temp](org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT)  Time elapsed: 0.12 s  <<< ERROR!
org.postgresql.util.PSQLException: 
ERROR: syntax error at or near "content"
  位置：44
        at org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT.assertExecuteQueryForPreparedStatement(GeneralDQLIT.java:85)
        at org.apache.shardingsphere.test.integration.engine.dql.GeneralDQLIT.assertExecuteQuery(GeneralDQLIT.java:60)



[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   GeneralDQLIT.assertExecuteQuery:58->assertExecuteQueryForStatement:71->BaseDQLIT.assertResultSet:79->BaseDQLIT.assertRows:104->BaseDQLIT.assertRow:122 
Expected: is "11"
     but: was "10"
[ERROR]   GeneralDQLIT.assertExecuteQuery:60->assertExecuteQueryForPreparedStatement:86->BaseDQLIT.assertResultSet:79->BaseDQLIT.assertRows:104->BaseDQLIT.assertRow:122 
Expected: is "11"
     but: was "10"
[ERROR]   GeneralDQLIT.assertExecute:97->assertExecuteForStatement:112->BaseDQLIT.assertResultSet:79->BaseDQLIT.assertRows:104->BaseDQLIT.assertRow:122 
Expected: is "11"
     but: was "10"
[ERROR]   GeneralDQLIT.assertExecute:99->assertExecuteForPreparedStatement:129->BaseDQLIT.assertResultSet:79->BaseDQLIT.assertRows:104->BaseDQLIT.assertRow:122 
Expected: is "11"
     but: was "10"
[ERROR] Errors: 
[ERROR]   GeneralDQLIT.assertExecuteQuery:58->assertExecuteQueryForStatement:70 » PSQL E...
[ERROR]   GeneralDQLIT.assertExecuteQuery:60->assertExecuteQueryForPreparedStatement:85 » PSQL
[ERROR]   GeneralDQLIT.assertExecute:97->assertExecuteForStatement:108 » PSQL ERROR: syn...
[ERROR]   GeneralDQLIT.assertExecute:99->assertExecuteForPreparedStatement:125 » PSQL ER...
[INFO] 
[ERROR] Tests run: 530, Failures: 4, Errors: 4, Skipped: 0
```

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dql/dql-integration-test-cases.xml
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dql/dql-integration-test-cases.xml
@@ -531,7 +531,7 @@
         <assertion parameters="1000:int, 100000:int" expected-data-source-name="read_dataset" />
     </test-case>
     
-    <test-case sql="SELECT user_id, CONCAT('SUM:', total, '.') content FROM (SELECT user_id, SUM(order_id_sharding) AS total FROM t_order_federate_sharding GROUP BY user_id HAVING SUM(order_id_sharding) > ?) AS temp" db-types="MySQL,PostgreSQL" scenario-types="tbl">
+    <test-case sql="SELECT user_id, CONCAT('SUM:', total, '.') AS content FROM (SELECT user_id, SUM(order_id_sharding) AS total FROM t_order_federate_sharding GROUP BY user_id HAVING SUM(order_id_sharding) > ?) AS temp ORDER BY temp.user_id" db-types="MySQL,PostgreSQL" scenario-types="tbl">
         <assertion parameters="1000:int" />
     </test-case>
     
@@ -555,7 +555,7 @@
         <assertion parameters="1100:int" />
     </test-case>
     
-    <test-case sql="SELECT user_id, SUM(order_id_sharding) FROM t_order_federate_sharding GROUP BY user_id HAVING SUM(order_id_sharding) > ?" db-types="MySQL,PostgreSQL" scenario-types="tbl">
+    <test-case sql="SELECT user_id, SUM(order_id_sharding) FROM t_order_federate_sharding GROUP BY user_id HAVING SUM(order_id_sharding) > ? ORDER BY user_id" db-types="MySQL,PostgreSQL" scenario-types="tbl">
         <assertion parameters="1000:int" />
     </test-case>
     

